java/client/src/main/java/glide/api/models/commands/RestoreOptions.java

```diff
@@ -11,9 +11,10 @@
 
 /**
  * Optional arguments to {@link GenericBaseCommands#restore(GlideString, long, byte[],
- * RestoreOptions)}
+ * RestoreOptions)}.
  *
  * @see <a href="https://valkey.io/commands/restore/">valkey.io</a>
+ * @apiNote <code>IDELTIME</code> and <code>FREQ</code> modifiers cannot be set at the same time.
```

Comment:
```suggestion
 * @apiNote <code>IDLETIME</code> and <code>FREQ</code> modifiers cannot be set at the same time.
```

Reply:
This isn't enforced anywhere else?  Ideally, we prevent the user from setting both at the same time through constructors.  Otherwise, we should throw a runtime error if both are defined before sending the request to Valkey.  
We should avoid letting Valkey process a command that we know will break.